pkg/operator2/oauth.go

```diff
@@ -96,7 +96,7 @@ func (c *authOperator) handleOAuthConfig(operatorConfig *authv1alpha1.Authentica
 				ServingInfo: configv1.ServingInfo{
 					BindAddress: fmt.Sprintf("0.0.0.0:%d", containerPort),
 					BindNetwork: "tcp4",
-					// we have valid certs provided by alfred so that we can use reencrypt routes
+					// we have valid serving certs provided by alfred so that we can use reencrypt routes
```

Comment:
Let's call alfred by his real name just so we don't confuse anyone not hip to the slang